framework/py/flwr/common/inflatable.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2025 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""InflatableObject base class."""
+
+
+import hashlib
+from typing import TypeVar
+
+from .constant import HEAD_BODY_DIVIDER
+
+T = TypeVar("T", bound="InflatableObject")
+
+
+class InflatableObject:
+    """Base class for inflatable objects."""
+
+    def deflate(self) -> tuple[bytes, str]:
+        """Deflate object."""
+        raise NotImplementedError()
+
+    @property
+    def object_id(self) -> str:
+        """Get object_id."""
+        _, object_id = self.deflate()
+        return object_id
+
+
+def get_object_id(object_content: bytes) -> str:
+    """Return a SHA-256 hash of the (deflated) object content."""
+    return hashlib.sha256(object_content).hexdigest()
+
+
+def get_object_body(content: bytes, cls: type[T]) -> bytes:
```

Comment:
Should we typically use `object_content` or just `content` when naming arguments/variables?